staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion_test.go

```diff
@@ -159,7 +159,7 @@ func TestWebhookConverter(t *testing.T) {
 			for i, checkFn := range test.checks {
 				name := fmt.Sprintf("check-%d", i)
 				t.Run(name, func(t *testing.T) {
-					ctc.setAndWaitStorageVersion(t, "v1beta2")
+					defer ctc.setAndWaitStorageVersion(t, "v1beta2")
```

Comment:
@jpbetz note this ^^: without the defer the storage version is undefined after the check have been run. Hence, the marker wait logic above might not do what we want.